index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/estilos-index.css">
+    <link rel="stylesheet" href="/css/flexboxgrid.min.css">
+    <title>CineOnline</title>
+</head>
+<body>
+<div class="container-1">
+        <header>
+            <div>
+                <nav class="row">
+                        <ul class="row col-xs-12 col-sm-12 col-md-12 col-lg-12">

```

Comment:
`row` class is added to the parent element, the style of the column element is made to reside inside a row element. `row` and columns classes should not be added to the same element. This could cause unexpected behavior.